tfx/examples/bert/utils/bert_tokenizer_utils.py

```diff
@@ -0,0 +1,170 @@
+# Lint as: python2, python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Prepressing using tensorflow_text BertTokenizer"""
+
+import tensorflow as tf
+import tensorflow_text as text
+from tensorflow.python.eager.context import eager_mode
+import tensorflow_hub as hub
+
+_CLS = '[CLS]'
+_PAD = '[PAD]'
+_SEP = '[SEP]'
```

Comment:
Are these necessary? If not, for my own education, can you explain how they're / going to use them?

(please disregard if you have a follow up commit which uses them directly).

Reply:
yeah of course. 
The reason we need to find the special tokens in the vocab file is that the vocab files for different Bert models are different.
Therefore, the indeices/token ids, of these special tokens are different. Therefore, we need to open the file and search for them.

And these special tokens are configurable in pre-train. i.e while preparing the vocab for a Bert pretrain, they can set anything as their special token. You could expect these tokens might be different in different languages.

Reply:
As for your previous comment: [cls] sentence A [sep] sentence B [sep]

the reference is figure 2 of the og bert paper
https://arxiv.org/pdf/1810.04805.pdf

Reply:
Gotcha, thanks for the clarification :)